sdk/communication/AzureCommunicationChat/Tests/ChatClientTests.swift

```diff
@@ -59,7 +59,7 @@ class ChatClientTests: XCTestCase {
         completionHandler: @escaping (String) -> Void
     ) {
         let participant = Participant(
-            id: id,
+            user: CommunicationUserIdentifier(identifier: id),
```

Comment:
This seems redundant. This would be a case where using `_` to omit the label would be acceptable. 
`init(_ identifier: String, ...)`